snippets/country-localization.liquid

```diff
@@ -31,11 +33,11 @@
             {% endif %}
             data-value="{{ country.iso_code }}"
           >
-            <span class="localization-form__currency">
-              {{- country.currency.iso_code }}
-              {{ country.currency.symbol }} |</span
-            >
-            {{ country.name }}
+            <span class="localization-form__disclosure__item-wrapper">
+              <span class="localization-form__country_flag"> {{ country | image_url: width: 14 | image_tag }} </span>
```

Comment:
```suggestion
              <span class="localization-form__country-flag"> {{ country | image_url: width: 14 | image_tag }} </span>
```

Reply:
Naming has been changed and will be visible in the next commit!